### PR TITLE
Use glossary tooltip for CSI

### PR DESCRIPTION
**Pull request by niranjandarshann**
Added Glossary Of CSi

### CODE REVIEW

**Comment by niranjandarshann** [comment]
Fixed issue https://github.com/kubernetes/website/issues/41904

**Comment by niranjandarshann** [comment]
@pohly  @xing-yang  
PTAL!

**Comment by tengqm** [comment]
> @pohly @xing-yang PTAL!

You may want to ping the reviewers if a PR has to be reviewed by them and the PR has been there for over a week or so. People in the community are super busy and most of them are multi-tasked. No one is supposed to review the PRs right away.

Be a little bit patient, even if you are fixing a critical bug that saves the world.

File: content/en/docs/concepts/storage/ephemeral-volumes.md [hunk 1]
```diff
@@ -47,8 +47,7 @@ different purposes:
   [secret](/docs/concepts/storage/volumes/#secret): inject different
   kinds of Kubernetes data into a Pod
 - [CSI ephemeral volumes](#csi-ephemeral-volumes):
-  similar to the previous volume kinds, but provided by special
-  [CSI drivers](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+  similar to the previous volume kinds, but provided by special {{< glossary_tooltip text="CSI" term_id="csi" >}} drivers
```
Comment (tengqm) [inline content/en/docs/concepts/storage/ephemeral-volumes.md:50]: I'm confused by this change. Is this really something that needs to be changed?
Or, you are just trying to find something to work on?
If it is the latter case, why don't you just browse through the issues list,
find one that has been triaged, and fix it?

Comment (sftim) [reply content/en/docs/concepts/storage/ephemeral-volumes.md:50]: (agreed - this doesn't look like a fix for https://github.com/kubernetes/website/issues/41904)

**Comment by sftim** [comment]
Sorry:
/retitle Use glossary tooltip for CSI

**Comment by niranjandarshann** [comment]
> > @pohly @xing-yang PTAL!
> 
> You may want to ping the reviewers if a PR has to be reviewed by them and the PR has been there for over a week or so. People in the community are super busy and most of them are multi-tasked. No one is supposed to review the PRs right away.
> 
> Be a little bit patient, even if you are fixing a critical bug that saves the world.

Ok I will take care of it. @tengqm

File: content/en/docs/concepts/storage/ephemeral-volumes.md (hunk 1 above)
Comment (niranjandarshann) [reply content/en/docs/concepts/storage/ephemeral-volumes.md:50]: > I'm confused by this change. Is this really something that needs to be changed? Or, you are just trying to find something to work on? If it is the latter case, why don't you just browse through the issues list, find one that has been triaged, and fix it?

Hi @tengqm my initial point of view for  the changes is redirecting to the [CSI drivers] (https://kubernetes-csi.github.io/docs/drivers.html) but as per the comment https://github.com/kubernetes/website/issues/41904#issuecomment-1625319900 i made the changes and raised the PR.

My motive toward the issue was :
CSI drivers are not redirecting to the correct link which should contain the list of the CSI drivers 

for which i given the **proposed solution**   we can add https://kubernetes-csi.github.io/docs/drivers.html link to redirect to the correct page which will be easy to understand.

Please have a look on it  and provide the suggestion  regarding this issue on which i can work on.

Comment (tengqm) [reply content/en/docs/concepts/storage/ephemeral-volumes.md:50]: IIUC, the original link was an intent to provide further details about the CSI driver concept. If that link becomes invalid, we may want to point to something else or just drop the link. Maybe we can replace the link target with something like `/docs/concepts/storage/volumes/#csi`, agree?

Comment (niranjandarshann) [reply content/en/docs/concepts/storage/ephemeral-volumes.md:50]: Thanks @tengqm for taking a look at this.
I made this change according to the comment on [issue #41904](https://github.com/kubernetes/website/issues/41904#issuecomment-1625319900).
if it doesn't seem good, can we change the hyperlink for the CSI driver to https://kubernetes-csi.github.io/docs/drivers.html, proposed solution as this page contains all the drivers that can be used with Kubernetes? I think it is more understandable to the readers.
WDYT?

Comment (niranjandarshann) [reply content/en/docs/concepts/storage/ephemeral-volumes.md:50]: > IIUC, the original link was an intent to provide further details about the CSI driver concept. If that link becomes invalid, we may want to point to something else or just drop the link. Maybe we can replace the link target with something like `/docs/concepts/storage/volumes/#csi`, agree?

Hi @tengqm 
It yes we can also consider this '/docs/concepts/storage/volumes/#csi` but the sentence having this issue is  talking about the CSI drivers so providing this https://kubernetes-csi.github.io/docs/drivers.html link will be easy to understand. 
WDYT?

**Comment by tengqm** [comment]
/lgtm
/approve
A glossary entry is acceptable.

**Pull request merged**